### DISCUSSION TITLE
Not possible to change entity id name in edit button node?

### DISCUSSION THREAD

**Discussion by platini76** [zachowj/node-red-contrib-home-assistant-websocket#1623] [Q&A]
hello please help me. I need to change default entity_id to my name.. but seems to be impossible.
where I am wrong?

<img width="904" alt="Screenshot 2024-10-02 alle 04 57 57" src="https://github.com/user-attachments/assets/af21dcdb-be97-4ecd-b86c-287ef01a63a5">

**Comment by zachowj** [comment]
Find the entity in Home Assistant, then use the "More Info" dialog to make changes directly.

![image](https://github.com/user-attachments/assets/e634b68d-564a-48cf-8f04-ff260dc8cfd5)

**Reply by juzi214032** [reply]
After I changed the entity id, this entity lost its association with the device

**Comment by platini76** [comment]
thanks
